test/tools/iotjs_build_info.js

```diff
@@ -21,8 +21,45 @@ if (process.env.IOTJS_ENV.indexOf("experimental") > -1)
 else
     stability = "stable"
 
+/* Check if certain es2015 features are available */
+var features = {}
+
+var hasPromise = (typeof Promise) != 'undefined';
```

Comment:
Wouldn't it be much simpler if `features` will be an array?

e.g.:
```js
var features = [];
if ((typeof Promise) != 'undefined') {
  features.push('Promise');
}
```

Reply:
@LaszloLango it could be an array, I just wanted to keep the output format consistent. The ``builtins`` property is an object, so I thought ``features`` should be an object as well, but I can change it if necessary

Reply:
It really depends on the python part (`testrunner.py`), so I am curious about the opinion of @galpeter. I saw in a previous review comment that we could use function of `set` in python like `contains`. I think if it is an array, than it it will be parsed as a set, but I am not sure.